test/Interop/Cxx/foreign-reference/Inputs/cxx-functions-and-methods-returning-frt.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
```

Comment:
Could you please add a test where a C++ that returns something that isn't a reference type is annotated with either of these attributes? e.g. a function that returns a plain C++ struct, or an enum.

Reply:
Such tests are already available undet the comments
// Testing Global/free C++ functions returning non-FRT
// Testing static functions returning non-FRT